### PR TITLE
Generate TARGETS file for inductor kernels in fbcode

### LINKED ISSUE #1097: Generate buck targets for inductor, inductor generated kernels are not runnable in fbcode atm
This needs to:

1) Redirect the compile cache somewhere TARGETS works
2) Produce a valid TARGETS file so we can buck run the generated py file
3) Probably get an e2e test in fbcode

### PR DESCRIPTION
**Pull request by bertmaher**
Fixes #1097

### CODE REVIEW

File: torchinductor/debug.py [hunk 1]
```diff
@@ -327,3 +327,23 @@ def graph_diagram(self, nodes: SchedulerNodeList):
 
     def output_code(self, filename):
         shutil.copy(filename, self.filename("output_code.py"))
+        V.debug.buck_targets()
+
+    def buck_targets(self):
+        with self.fopen("TARGETS") as fd:
+            fd.write("""
```
Comment (jansel) [inline torchinductor/debug.py:334]: Suggest `textwrap.dedent()` so you can indent this a bit nicer.

**Review by jansel — COMMENTED** [review]
`make format` should fix lint issues

**Review by voznesenskym — CHANGES_REQUESTED** [review]
We are missing the C++ side of things here (Headers, .so). I am also not sure we generate a single file. a `glob` rule for the out dir may cover things better.

Where is it writing TARGETS to atm?

**Comment by jansel** [comment]
We have migrated `torchdynamo` to `torch._dynamo` and will use the `pytorch/pytorch` repo for future development.  Please resubmit this PR to https://github.com/pytorch/pytorch/

More details and instructions to port this PR over can be found in https://github.com/pytorch/torchdynamo/issues/1588

**Pull request closed without merging**